package.json

```diff
@@ -50,17 +50,17 @@
     "@babel/plugin-transform-runtime": "^7.11.5",
     "@babel/preset-react": "^7.10.4",
     "@sap-theming/theming-base-content": "~11.1.23",
-    "@storybook/addon-a11y": "6.0.28",
-    "@storybook/addon-actions": "6.0.28",
-    "@storybook/addon-cssresources": "6.0.28",
-    "@storybook/addon-docs": "6.0.28",
-    "@storybook/addon-knobs": "6.0.28",
-    "@storybook/addon-links": "6.0.28",
-    "@storybook/addon-storyshots": "6.0.28",
-    "@storybook/addon-toolbars": "6.0.28",
-    "@storybook/addon-viewport": "6.0.28",
-    "@storybook/addons": "6.0.28",
-    "@storybook/html": "6.0.28",
+    "@storybook/addon-a11y": "6.1.6",
+    "@storybook/addon-actions": "6.1.6",
+    "@storybook/addon-cssresources": "6.1.6",
+    "@storybook/addon-docs": "6.1.6",
+    "@storybook/addon-knobs": "6.1.6",
+    "@storybook/addon-links": "6.1.6",
+    "@storybook/addon-storyshots": "6.1.6",
+    "@storybook/addon-toolbars": "6.1.6",
+    "@storybook/addon-viewport": "6.1.6",
+    "@storybook/addons": "6.1.6",
+    "@storybook/html": "6.1.6",
```

Comment:
This one is currently broken in ie11 fyi - https://github.com/storybookjs/storybook/issues/13272